### PR TITLE
[PagePartBundle] Fix for PHP7.2 - value passed to count() cannot be null in PagePartAdmin#persist

### PR DESCRIPTION
**Pull request by tentwofour**
Added cast to array to ensure regions without page parts pass an empty array to the loop, rather than null value (throws fatal ContextErrorException: Warning: count(): Parameter must be an array or an object that implements Countable)

| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets |

### CODE REVIEW

**Comment by sandergo90** [comment]
@tentwofour can you update your PR to meet the contribution guidelines? Thanks

File: src/Kunstmaan/PagePartBundle/PagePartAdmin/PagePartAdmin.php [hunk 1]
```diff
@@ -265,7 +265,7 @@ public function persist(Request $request)
         $ppRefRepo = $this->em->getRepository('KunstmaanPagePartBundle:PagePartRef');
 
         // Add new pageparts on the correct position + Re-order and save pageparts if needed
-        $sequences = $request->get($this->context.'_sequence');
+        $sequences = (array) $request->get($this->context.'_sequence');
```
Comment (sandergo90) [inline src/Kunstmaan/PagePartBundle/PagePartAdmin/PagePartAdmin.php]: You can add a default value to the request->get method

**Comment by sandergo90** [comment]
@tentwofour the master branch is also used for new features. 

https://github.com/Kunstmaan/KunstmaanBundlesCMS/blob/master/docs/99-02-pull-requests.md

**Comment by KevinSleegers** [comment]
@sandergo90 Could this also be fixed for Kunstmaan 4.2? Getting the same error when using that version with PHP7.2.

**Comment by sandergo90** [comment]
@KevinSleegers Bug fixes are normally done for the latest minor version branch. So this would be the 5.0.4 branch. It's best that you upgrade to the 5.0 branch for this.

**Comment by tentwofour** [comment]
The rust is showing after 5 months away :) Changed the cast to spec'ing a default value in the Request#get call, and changed the commit message...reread the PR guidlines, something else I've missed?

**Comment by sandergo90** [comment]
@tentwofour hi ! Maybe change the array() syntax to the short syntax

**Comment by sandergo90** [comment]
You should also rebase against 5.0 branch and change the base of the PR.

**Comment by Devolicious** [comment]
Hi @tentwofour,

I've updated this PR to be submitted to branch 4.1 since we have to release another important bugfix. Could you please rebase this PR (and maybe change the `array()` to short syntax) so we are able to merge it?

Thx!

**Pull request closed without merging**